mysqld_exporter.go

```diff
@@ -107,6 +111,41 @@ var (
 		"The total time of index I/O wait events for each index and operation.",
 		[]string{"schema", "name", "index", "operation"}, nil,
 	)
+	performanceSchemaEventsStatementsDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, performanceSchema, "events_statements_total"),
+		"The total count of events statements by digest",
```

Comment:
Some of the metric descriptions end with a period, others don't. It'd be great to make that consistent. (I prefer with period FWIW)